app/views/contexts/submit_questionnaire_context.py

```diff
@@ -53,7 +53,7 @@ def _get_summary_context(
         }
 
     def _build_all_groups(self) -> Generator[dict, None, None]:
-        """ NB: Does not support repeating sections """
+        """ NB: Does not support repeating sections"""
```

Comment:
This change looks unrelated to the pull request and should be removed.

Reply:
Done - apologies - this was VS Code using the incorrect formatter after an update